ledger/ledger-on-memory/src/test/suite/scala/com/daml/ledger/on/memory/InMemoryLedgerReaderWriterIntegrationSpec.scala

```diff
@@ -61,5 +62,6 @@ class InMemoryLedgerReaderWriterIntegrationSpec
       readerWriter,
       metrics,
       enableSelfServiceErrorCodes = true,
+      NotSupportedDeduplicationPeriodConverter,
```

Comment:
Why can't we support it here?

Reply:
We basically test one layer too low at this point, therefore we don't have a way of reading a completion at this level.